src/app/kpis/page.tsx

```diff
@@ -58,49 +58,119 @@ function LoadingState() {
+              <p className="text-[14px] text-[#768089]">
+                TOTAL Trade Volume (XRD)
+              </p>
+              <p>{displayNumber(kpiData.tradeVolume.total.XRD)}</p>
+            </div>
+            <div className="border-[#3c3d3d] border-[2px] p-4 rounded-xl mt-1">
+                <LineChart
+                title={"Weekly Trade Volume (XRD)"}
+                amount={kpiData.tradeVolume.total.XRD}
+                x={kpiData.tradeVolume.weekly.XRD.map((o) => o.weekIdentifier)}
+                y={kpiData.tradeVolume.weekly.XRD.map((o) => o.value)}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col mt-8">
+        <p className="text-[24px] m-1">Website</p>
+        <div className="flex">
+          <div className="border-[#3c3d3d] border-[2px] p-4 rounded-xl m-1">
+            <LineChart
+              title={"Weekly Website Page Requests"}
+              amount={55467}
+              x={kpiData.website.pageRequests.map((o) => o.weekIdentifier)}
+              y={kpiData.website.pageRequests.map((o) => o.value)}
+            />
+          </div>
+          <div className="border-[#3c3d3d] border-[2px] p-4 rounded-xl m-1">
+            <LineChart
+              title={"Weekly Website Unique Visitors"}
+              amount={6452}
+              x={kpiData.website.uniqueVisitors.map((o) => o.weekIdentifier)}
+              y={kpiData.website.uniqueVisitors.map((o) => o.value)}
+            />
+          </div>
+        </div>
+      </div>
+      <div className="flex flex-col mt-8 w-full">
+        <p className="text-[24px]">Socials</p>
+        <div className="flex w-full justify-between">
+            <div className="flex w-[33%] border-[#3c3d3d] border-[2px] p-4 rounded-xl">
+            <img src="/socials/youtubu.png" width={56} height={56} alt="" />
+            <div className="pl-2">
+              <p className="text-[11px]">Subscribers</p>
+              <p className="text-2xl text-[#FFFFFF]">{kpiData.socials.youtubeSubscribers}</p>
+            </div>
+          </div>
+            <div className="flex w-[33%] border-[#3c3d3d] border-[2px] p-4 rounded-xl">
+            <img src="/socials/instagram.png" width={56} height={56} alt="" />
+            <div className="pl-2">
+              <p className="text-[11px]">Followers</p>
+              <p className="text-2xl text-[#FFFFFF]">{kpiData.socials.instagramFollowers}</p>
+            </div>
+          </div>
+            <div className="flex w-[33%] border-[#3c3d3d] border-[2px] p-4 rounded-xl">
+            <img src="/socials/twitter.png" width={56} height={56} alt="" />
+            <div className="pl-2">
+              <p className="text-[11px]">Followers</p>
+              <p className="text-2xl text-[#FFFFFF]">{kpiData.socials.twitterFollowers}</p>
+            </div>
+          </div>
+        </div>
+        </div>
+      </div>
     </>
   );
 }
 
 function LineChart({
   title,
+  amount,
```

Comment:
Please remove `amount`. It can be calculated on the spot by adding all y-values, e.g. 
```
y.reduce((a,b) => a+b)
```